python/paddle/fluid/dygraph/__init__.py

```diff
@@ -44,6 +44,9 @@
 from . import jit
 from .jit import *
 
+from . import static_runner
+from .static_runner import *
```

Comment:
Do we really need to import all? I mean, do we only need `StaticModelRunner` ?

Reply:
done